eng/common/tools.ps1

```diff
@@ -743,8 +763,8 @@ function MSBuild() {
       (Join-Path $basePath (Join-Path netcoreapp2.1 'Microsoft.DotNet.Arcade.Sdk.dll'))
       (Join-Path $basePath (Join-Path netcoreapp3.1 'Microsoft.DotNet.ArcadeLogging.dll')),
       (Join-Path $basePath (Join-Path netcoreapp3.1 'Microsoft.DotNet.Arcade.Sdk.dll'))
-      (Join-Path $basePath (Join-Path net8.0 'Microsoft.DotNet.ArcadeLogging.dll')),
-      (Join-Path $basePath (Join-Path net8.0 'Microsoft.DotNet.Arcade.Sdk.dll'))
+      (Join-Path $basePath (Join-Path net7.0 'Microsoft.DotNet.ArcadeLogging.dll')),
+      (Join-Path $basePath (Join-Path net7.0 'Microsoft.DotNet.Arcade.Sdk.dll'))
```

Comment:
This resulted in the error `##[error](NETCORE_ENGINEERING_TELEMETRY=Build) Unable to find arcade sdk logger assembly.` It seems not correct.

Reply:
This caused the build failure as mentioned above using stage1 bootstrap. Refer to https://dev.azure.com/dnceng-public/public/_build/results?buildId=343349&view=logs&j=20125c81-7edb-551b-693a-61efae016b74&t=7ce25265-aabe-59d3-d73c-06619cfcc3c4&l=128.

Reply:
Updating build tool framework to net8.0 when buiding with dotnet could resolve the problem.

Reply:
It seems to be working now--I think because `$buildTool.Framework` is `net8.0` so lines 760/761 do the job.

Reply:
Yes, you are right.